common/arkimeUtil.js

```diff
@@ -71,6 +71,15 @@ class ArkimeUtil {
     return next();
   }
 
+  // ----------------------------------------------------------------------------
+  static noCache (req, res, ct) {
```

Comment:
Isn't this already in ViewerUtils? We should remove it from there and use this everywhere

Reply:
yep, was going to do another PR for that